client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2043,6 +2043,21 @@ Undocumented.prototype.startExport = function( siteId, advancedSettings, fn ) {
 };
 
 /**
+ * Start an export from a Jetpack site
+ *
+ * @param {int}       siteId            The site ID
+ * @param {Object}    advancedSettings  Advanced export configuration
+ * @param {Function}  fn                The callback function
+ * @returns {Promise}                   A promise that resolves when the export started
+ */
+Undocumented.prototype.startJetpackExport = function( siteId, advancedSettings, fn ) {
+	return this.wpcom.withLocale().req.post( {
+		apiVersion: '1.1',
+		path: `/sites/${ siteId }/export`
```

Comment:
/exports/start is part of a set of wpcom-only endpoints used to activate the wpcom async. exporter. (Along with exports/${ exportId }.

The export process on .org servers via Jetpack is a sync. operation. The /export endpoint is a Jetpack-only endpoint that handles export on the .org side and returns the downloadURL in one call. This endpoint is newly created by me for this project so I doubt @jordwest knows it.

I created a separate endpoint with a different path for the .org site because of the different nature of the process (sync. vs async.) and the similarity to the original export URL: wp-admin/export.php